news/WildFly-s2i-19-released.adoc

```diff
@@ -0,0 +1,148 @@
+= WildFly 19 S2I image has been released on quay.io
+jfdenise
+2020-03-20
+:awestruct-tags: [wildfly, galleon]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+
+==  WildFly 19 S2I docker images
+
+The WildFly s2i builder and runtime Docker images for WildFly 19 have been released on link:https://quay.io/organization/wildfly[quay.io/wildfly].
```

Comment:
is it s2i or S2I? The first time it is mentioned we could maybe remind that it means (Source-to-Image)